microshift_storage/persistent_storage_microshift/microshift-storage-plugin-overview.adoc

```diff
@@ -0,0 +1,10 @@
+:_content-type: ASSEMBLY
+[id="microshift-storage-plugin-overview"]
+= MicroShift storage plugin overview 
+include::_attributes/common-attributes.adoc[]
+:context: persistent-storage-csi
+
+toc::[]
+
+
```

Comment:
I know you told me already that this has no content yet and that's expected, but again just saying so here :)

Reply:
Next PR :D